sdk/identity/azure-identity/samples/code_snippets.py

```diff
@@ -0,0 +1,57 @@
+# ------------------------------------
```

Comment:
Let's use more specific name for the file. (feel free to split the snippets into different files).

And update https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/samples/README.md

Reply:
Sure, how about `credential_creation_code_snippets.py`. I think, since these are very basic and primarily intended for use in docstrings, keeping these consolidated in one file is fine. Eventually, we can add more as it'd be nice to have basic code snippets in all our credential docstrings showcasing how to instantiate the various credentials.